addons/udes_delivery_control/models/picking_extra_fields.py

```diff
@@ -0,0 +1,147 @@
+from odoo import api, fields, models, _
+from ..misc import date_diff, validate_dates
+
+
+class StockPickingExtraFields(models.Model):
+    """
+        Model to hold all extra fields of stock.picking object
+    """
+
+    _name = "udes_picking_extras"
```

Comment:
Similar to the model comment, I think this should be `stock.picking.delivery_control` or similar.

Reply:
In the future we might move this to different module and when that happens we don't want to break anything with current setup. Moving to different module after and renaming the table name would be bad, so I name it as generic as I can.

Reply:
If you want to keep it generic then I think `_name` should be `udes.stock.picking.extras` for consistency.

Reply:
done